tasks/install-rhel.yml

```diff
@@ -5,6 +5,5 @@
 
 - name: install foreman rubygem-apipie-bindings package
   yum: name=rubygem-apipie-bindings state=present
-  when: foreman_proxy_register  
```

Comment:
Yeah I got that you're proposing install-rhel.yml and install-ubuntu.yml
but
I was talking about your changes **within** file itself
If you open [you copy](https://github.com/kireevco/ansible-role-foreman-proxy/blob/master/tasks/install-rhel.yml) in one tab and my [initial version](https://github.com/kostyrevaa/ansible-role-foreman-proxy/blob/master/tasks/install.yml) in the other
you'll see no difference in code itself
you just added one empty line